less/index.less

```diff
@@ -52,8 +52,209 @@ table {
 * {
     box-sizing: border-box;
 }
+/**** Variables ****/
+
+@defaultbg: #808080;
+
+/**** Variables ****/
+
+/***** Mixins ****/
+
+.center() {
+	display: flex;
+	justify-content: center;
+  }
+
+  .normalize(@decoration, @color) {
+	text-decoration: @decoration;
+	color: @color;
+  }
+
+/**** Mixins *****/
+
 
 html, body {
-    height: 100%;
+	height: 100%;
+	font-family: 'Titillium Web', sans-serif;
+	font-size: 62.5%;
+	background-color: black;
+
+}
+
+.container {
+	max-width: 880px;
+	width: 100%;
+	margin: 0 auto;
+	padding: 0 2.5%;
+	background-color: white;
+}
+
+header {
+	width: 100%;
+	display: flex;
+}
+
+
+
+p {
+	font-size: 1.4rem;
+	line-height: 1.5;
+}
+
+a {
+	font-size: 1.4rem;
+	line-height: 1.5;
+	
+}
+
+h1 {
+	font-size: 4rem;
+	margin-top: 13px;
+	margin-bottom: 13px;
+}
+
+h2 {
+    font-size: 1.5rem;
+}
+
+h3 {
+	font-size: 3rem;
+}
+
+h4 {
+    font-size: 2rem;
+}
+
+li {
+	font-size: 1.4rem;
+}
+
+// button { ----- Removed this feature from site ------
+// 	background-color: @defaultbg;
+// 	border-radius: 6px;
+//     height: 37px;
+//     width: 14.9%;
+//     font-family: inherit;
+//     font-size: 15px;
+// 	margin-top: 15px;
+// 	color: white;
+// 	margin-bottom: 65px;
+// } ----- Removed this feature from site ------
+
+
+nav {
+	margin-left: 63%;
+	width: 52%;
+	display: flex;
+	justify-content: space-between;
+	margin-top: 20px;
+
+	a {
+		text-decoration: none;
+		color: black;
+		font-weight: bold;
+	}
+}
+
+.jtpic {
+	width: 25%;
+}
+
+.cta {
+	background-color: #F1F1F1;
+	width: 100%;
+	padding-left: 3%;
+	padding-top: 44px;
+	margin-top: 22px;
+	border-radius: 4px;
+
+	.cta-text {
+		border-bottom: 1px solid lightgray;
+		margin-bottom: 18px;
+		margin-right: 3%;
+	}
+}
+
+.contact {
+	border-top: 1px solid lightgray;
+	margin-right: 3%;
+	margin-top: 14px;
+	padding-top: 20px;
+	padding-bottom: 20px;
 }
 
+.main-content {
+	display: flex;
+	flex-direction: column;
+	margin-top: 28px;
+
+	.top-content {
+		display: flex;
+		.center;
+
+		.Bio {
+			margin-right: 4.5%;
+		}
+
+		.Bio h3 {
+			padding-bottom: 6px;
+
+		}
+	}
+
+}	
+
+	.Skills {
+		padding-left: 2.1%;
+		width: 286%;
+
+		.skills-list {
+			border: 1px solid black;
+			margin-top: 9px;
+			font-weight: bold;
+
+			a {
+			text-decoration: none;
+			color: black;
+			display: block;
+			border-bottom: 1px solid black;
+			text-align: start;
+			margin-top: 6px;
+			padding-left: 15px;
+			padding-bottom: 5px;
+}
+	
+	}
+}
+
+	
+.bottom-content {
+	margin-top: 28px;
+}
+
+.bottom-content .work-history div {
+	border-bottom: 1px solid lightgray;
+}
+
+		.bottom-content .work-history h4 {
+			border-bottom: 1px solid lightgray;
+			font-size: 2rem;
+			padding-bottom: 7px;
+}
+
+		.Apple {
+			padding-bottom: 20px;
+			padding-top: 20px;
+
+
+				ol {
+				padding-top: 16px;
+			}
+		}
+
+
+.footer {
+	width: 100%;
+	background: gray;
+	
+}
```

Comment:
overall everything looks great. Your commit history is solid and your project looks great. I appreciate the little things you added too lol. Nice touch!